database/dynamodb/new.go

```diff
@@ -47,3 +47,7 @@ func NewDynamoDB(conf config.DynamoDB) (*DynamoDB, error) {
 func (db *DynamoDB) Init() error {
 	return db.createTables()
 }
+
+func (db *DynamoDB) Close() {
+	//db.client.Close()
```

Comment:
Why commented?

Reply:
Was there from dev, when assumed there was a dynamo close method. There doesn't appear to be one, so the comment has been removed.